exporter/mezmoexporter/factory.go

```diff
@@ -46,17 +47,22 @@ func createDefaultConfig() config.Exporter {
 }
 
 // Create a log exporter for exporting to Mezmo
-func createLogsExporter(ctx context.Context, settings component.ExporterCreateSettings, exporter config.Exporter) (component.LogsExporter, error) {
-	if exporter == nil {
+func createLogsExporter(ctx context.Context, settings component.ExporterCreateSettings, exporterConfig config.Exporter) (component.LogsExporter, error) {
+	log := settings.Logger
+	log.Info("creating Mezmo logs exporter")
```

Comment:
Why is this important? We print when se start components.

Reply:
I took inspiration from https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0dd3a7f29c05605e82b4017e280d82a4cc4c202e/exporter/datadogexporter/factory.go#L147

I'll remove it.

Reply:
@mx-psi any idea why?

Reply:
I think this can be removed both here and in the Datadog exporter, it's not a very useful log.

Reply:
Removed them from the Datadog exporter on #11028

Reply:
This request was resolved and everything else appears to be in order, so I am resolving this comment and dismissing the blocking review in order to merge.